gpustat/cli.py

```diff
@@ -93,15 +94,18 @@ def main(*argv):
                         help='Display PID of running process')
     parser.add_argument('-F', '--show-fan-speed', '--show-fan',
                         action='store_true', help='Display GPU fan speed')
-    parser.add_argument('--json', action='store_true', default=False,
-                        help='Print all the information in JSON format')
-    parser.add_argument('-v', '--version', action='version',
-                        version=('gpustat %s' % __version__))
+    parser.add_argument(
```

Comment:
How do you think tying both enc and dec into one bool arg like `show-codec`?

Reply:
oh, that's  what I already have, but it's called `--show-encoder` and takes a parameter (defaults to both enc and dec). Good call with the naming change though, I'll change it to `--show-codec`

Reply:
Renamed it to --show-codec